timeseries/tests/unittests/models/test_gluonts.py

```diff
@@ -145,19 +145,19 @@ def test_when_prophet_model_saved_then_prophet_parameters_are_loaded(growth, n_c
 def test_when_hyperparameter_tune_called_on_prophet_then_hyperparameters_are_passed_to_underlying_model(
```

Comment:
Good question and the answer is no. The `hpo_results` are different and we need different unit tests for that. I plan to add unit tests for ray backend with timeseries when we enable it. Currently, we still use our custom backend. This PR is more of a interface update for timeseries

Reply:
So I think that's OK from the time series side and we can address later. But just for my info: depending on the backend `hpo_results` has a different schema? So we're not really unifying the API for HPO, just unifying how HPO is invoked?

Reply:
Our custom logic will return the results as a dict while ray will return it as a custom object. I haven't unified the interface because we don't really use the hpo_results right now. We might want to discuss on what interface we need before I work on unifying them, but that's definitely on the todo list